docs/Installation/Installation_Guide.rst

```diff
@@ -1,579 +1,581 @@
+
+3. On your cluster, open port 6443 for remote access to cluster services.
+
+4. Run ``kubectl proxy`` from your local machine. This opens a tunnel to
+   the Kubernetes cluster. While this tunnel is up and running, the following
+   tools are available:
+
+   +-------------------------+---------------------------------------------------------+-------------------------------------------------------------------------------------------------+---------------------------------------+
+   | Service                 | Role                                                    | Link                                                                                            | Notes                                 |
+   +=========================+=========================================================+=================================================================================================+=======================================+
+   | `Kubernetes dashboard`_ |A general purpose, web-based UI for Kubernetes clusters  | http://localhost:8001/api/v1/namespaces/kube-system/services/https:kubernetes-dashboard:/proxy/ | Username: kube                        |
+   |                         |                                                         |                                                                                                 |                                       |
+   |                         |                                                         |                                                                                                 | Password: See inventory/quickstart-   |
+   |                         |                                                         |                                                                                                 | cluster/credentials/kube_user.creds   |
+   |                         |                                                         |                                                                                                 | in the Kubernetes host.               |
+   +-------------------------+---------------------------------------------------------+-------------------------------------------------------------------------------------------------+---------------------------------------+
+   | `Grafana`_              | Monitoring dashboards for cluster services              | http://localhost:8001/api/v1/namespaces/kube-ops/services/kube-prometheus-grafana:http/proxy/   |                                       |
+   +-------------------------+---------------------------------------------------------+-------------------------------------------------------------------------------------------------+---------------------------------------+
+   | `Cerebro`_              | An administration and monitoring console for            | http://localhost:8001/api/v1/namespaces/kube-ops/services/cerebro:http/proxy/                   | When accessing Cerebro, connect it to |

```

Comment:
We pre-provision a config.

Reply:
How would you have this reflected in the documentation? Is Cerebro preconfigured to elasticsearch:9200?

Reply:
Following precedent in ZENKO-921 (Kibana, next comment) and deleting setup note.

Reply:
Yeah, there's an entry in a pre-populated list of 'known clusters' when you open Cerebro.

Reply:
Done.